server/presenters/share.js

```diff
@@ -5,9 +5,11 @@ import { presentUser } from ".";
 export default function present(share: Share) {
   return {
     id: share.id,
+    documentId: share.documentId,
     documentTitle: share.document.title,
     documentUrl: share.document.url,
-    url: `${process.env.URL}/share/${share.id}`,
+    published: share.published,
+    url: `${share.team.url}/share/${share.id}`,
```

Comment:
fixes: Share links should include team's subdomain to allow authentication to be read if the user is signed in.